### ISSUE TITLE
Problem with verifying Windows host binaries

### ISSUE THREAD

**Issue by bkarrlander** [Azure/iotedge-eflow#79]
**Question Summary**
In the deploy phase of the eflow setup, we get a warning message when verifying Windows host binaries saying "Missing issuer" followed by an error saying "Signer is not trusted". What could be the problem? Is a specific firewall opening required for verifying the binaries?

**Detailed background**
We are using version 1.1.2108.0.

We have configured the machine's firewall and ports according to "https://docs.microsoft.com/en-us/azure/iot-edge/troubleshoot?view=iotedge-2020-11#check-your-firewall-and-port-configuration-rules" but it seems not to be enough. 

Full error message:  

```
[09/23/2021 12:28:10] Verifying installation

 - Verifying expected Windows host binaries
WARNING: Chain status:
    CERT_TRUST_REVOCATION_STATUS_UNKNOWN
    CERT_TRUST_IS_OFFLINE_REVOCATION
    CERT_TRUST_IS_PARTIAL_CHAIN
Missing issuer: CN=Microsoft Code Signing PCA 2011, O=Microsoft Corporation, L=Redmond, S=Washington, C=US

[09/23/2021 12:28:26] Exception caught!!!

- Signer is not trusted (file: C:\Program Files\Azure IoT Edge\wssdagent.exe, certificate Subject: CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US)
```

![eflow-problem](https://user-images.githubusercontent.com/10998606/134498849-2d682341-2ad2-4145-a31a-5e254f7e5f68.png)

**Comment by TerryWarwick** [comment]
@bkarrlander,

TL;DR - You may have an expired certificate or broken trust chain for your certificates.  Run Window Update until it does not offer any additional updates to ensure that your certificates are current and try again.

This condition typically occurs when the trust chain of the certificates on your system is broken or certificates have expired.   

1. To check the certificate chain is intact, please go to Start and type "_Manage Computer Certificates_" and select Open next to the matching application.
2. Locate "_Microsoft Code Signing PCA 2011_" under **Trusted Publishers\Certificates**.
3. Check expiration date
4. Open the certificate and confirm that the description immediately below Certificate Information **does not** state something like "_Windows does not have enough information to verify this certificate._"
5. If it does, please run Windows Update to make sure that all updates have been applied to the system.  Certificates are regularly updated by Windows Update to prevent this situation.
6. This can also be caused by disabling Automatic Certificate Updates in Group Policy - Additional info, see http://technet.microsoft.com/en-us/library/cc734054(WS.10).aspx

**Comment by bkarrlander** [comment]
Thanks for your help
